projects/ion/src/lib/chip/chip.component.html

```diff
@@ -0,0 +1,7 @@
+<button (click)="select()" [ngClass]="{
+    chip: !selected,
+    'chip-selected': selected,
+    'chip-md': size === 'md',
+    'chip-disabled' : disabled}">
```

Comment:
```suggestion
>
```

Reply:
Add [disabled]="disabled" in HTML tag and inner CSS add button[disabled] { style.... }